testkit/tests/service_hooks/main.rs

```diff
@@ -35,6 +37,10 @@ fn test_handle_commit() {
     for i in 1..5 {
         testkit.create_block();
         let tx = TxAfterCommit::new_with_signature(Height(i), &Signature::zero());
-        assert!(testkit.mempool().contains_key(&tx.hash()));
+        testkit.add_tx(tx.raw().clone());
```

Comment:
Sorry, I don't get it. Isn't this cheating? Afaik, the test should check that `tx` appears in the mempool *without* manually adding it there.